### PR TITLE
Reply to thread

### PR DESCRIPTION
**Pull request by mixmix**
not working yet.

Two bugs I'm wrestling with: 
- threads are rendering wrong (it's rending a subsection of the thread)
- replies are not being sent, but no error

### CODE REVIEW

File: message/html/compose.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        if (typeof prepublish === 'function') {
+          content = prepublish(content)
+        }
+      } catch (err) {
+        publishBtn.disabled = false
+        if (cb) cb(err)
+        else throw err
+      }
+
+      return api.message.async.publish(content, done)
```
Comment (mixmix) [inline message/html/compose.js]: hey @dominictarr publishing messages is badly blocked. 
I've copied this code over from patchbay, and commented out a bunch of the channel related stuff which isn't needed for private threads (what I've started building).

This call here takes an appropriate looking content object, and `message.async.publish` just passes directly to `sbot.async.publish`  in patchcore - see `/sbot.js`.

What's killing me is that there are no errors when I step through ... and no return value. The message is encrypted and is ready for publishing then is gone.
Would love to work through this together

**Comment by mixmix** [comment]
@dominictarr this is ready for a smoke test and quick review

**_Please merge when if you're happy with it_**

**Comment by dominictarr** [comment]
I checked out 6ec0db7801b46b66f52d44d1d0a6006fdc71b53f
but do not see any textbox???

**Comment by dominictarr** [comment]
false alarm, I hadn't fetched the correct code.

**Comment by mixmix** [comment]
thanks ! :cocktail:

**Pull request merged**